docker/openproblems-r-base/Dockerfile

```diff
@@ -35,16 +19,7 @@ COPY ./scripts/install_renv.R ./
 COPY ./docker/openproblems-r-base/r_requirements.txt ./r_requirements.txt
 RUN R -e "source(\"install_renv.R\"); install_renv(\"r_requirements.txt\")"
 
-# Install kernelspec
-RUN R -e "IRkernel::installspec(user = FALSE);"
-
```

Comment:
This is not a sagemaker requirement, but an rpy2 requirement i think.

Reply:
Nope, that's sagemaker (allows R kernels inside jupyter). Tests are passing, so looks like it's not required :)

Reply:
i guess i'm mixing up running r magics in jupyter with running R kernels via rpy2 in jupyter.